docs/howto/use_modules_and_layers.md

```diff
@@ -0,0 +1,270 @@
+---
+id: use_layers
+title:  "Use modules and layers"
+---
+
+# Unleash ZIO environment with `ZLayer`
+
+`ZIO` is designed around 3 parameters, `R, E, A`. `R` represents the _requirement_ for the effect to run, meaning we need to fulfill
+the requirement in order to make the effect _runnable_. We will dedicate this post to explore what we can do with `R`, as it plays a crucial role in `ZIO`.
```

Comment:
I think I would use something other than "post" here since this is going to be part of the permanent documentation.

Reply:
Yes, you are right. This started as a post then it became documentation 👍